Dockerfile

```diff
@@ -85,6 +85,7 @@ RUN cd \
     && cd opencv-3.2.0/build \
     && cmake -DOPENCV_EXTRA_MODULES_PATH=../../opencv_contrib-3.2.0/modules/ .. \
     && make -j8 \
+    && make install \
```

Comment:
I'm not at my work machine, but I don't see how install could make things worse, so will accept the pull requests